Makefile

```diff
@@ -0,0 +1,11 @@
+# Arcane incantation to print all the other targets, from https://stackoverflow.com/a/26339924
+help:
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
```

Comment:
MinGW didn't solve the issue with "make" command
There is a Makefile Tools in VS code extensions which might help...I'll test it and let you know if it works